client/src/App.jsx

```diff
@@ -7,11 +7,12 @@ import 'semantic-ui-css/semantic.min.css';
 import Layout from './components/hoc/Layout/Layout';
 import Home from './containers/Home/Home';
 import BlogBox from './components/blogBox';
-import Authentication from './containers/Authentication';
+import Authentication from './containers/Authentication/Authentication';
 import * as actions from './store/actions/index';
 import Loader from './components/UI/Loader';
-import Category from "./containers/Category";
-import CategoryType from "./containers/CategoryType";
+import Category from "./containers/Category/Category";
```

Comment:
sam here

Reply:
will do refactoring in last on all the components after done developing.